pkg/runtime/testdata/outputs/bundle-outputs-error.yaml

```diff
@@ -3,8 +3,7 @@ mixins:
 
 name: outputs
 version: 0.1.0
-invocationImage: porter-outputs:latest
-tag: deislabs/porter-outputs-bundle:v0.1.0
+tag: getporter/porter-outputs-bundle:v0.1.0
```

Comment:
nit: let's make sure to do a sweep for *-bundle and clean that up, just to help remove a bad practice that we don't want anyone to see and repeat.